sematic/ui/src/components/Notes.tsx

```diff
@@ -7,9 +10,8 @@ import UserAvatar from "./UserAvatar";
 export function NoteView(props: {
   note: Note;
   author: User;
-  onRootIdChange: (rootId: string) => void;
 }) {
-  const { note, author, onRootIdChange } = props;
```

Comment:
Please confirm you performed these tests:

1. When submitting a new resolution, click the "View" link on the new resolution notification popup as soon as it appears in the UI, and confirm that the page correctly updates as the runs are created.

2. Switch to a resolution page using the drop-down "Run" selector. Switch to another one and confirm the details are correctly loaded and rendered.

3. While viewing a run details page, click on another run in the same resolution and confirm the details are correctly loaded and rendered.

4. Do the above for both detached and attached resolutions.

Reply:
Hi @tscurtu, Thanks for mentioning these test cases. They are very pertinent to this PR.

I have tested those with both detached and attached resolutions. They all pass for me.

Reply:
Thank you!